server/internal/usecase/interfaces/item.go

```diff
@@ -4,20 +4,22 @@ import (
 	"context"
 	"errors"
 
+	"github.com/nicksnyder/go-i18n/v2/i18n"
 	"github.com/reearth/reearth-cms/server/internal/usecase"
 	"github.com/reearth/reearth-cms/server/pkg/id"
 	"github.com/reearth/reearth-cms/server/pkg/item"
 	"github.com/reearth/reearth-cms/server/pkg/key"
 	"github.com/reearth/reearth-cms/server/pkg/model"
 	"github.com/reearth/reearth-cms/server/pkg/schema"
 	"github.com/reearth/reearth-cms/server/pkg/value"
+	"github.com/reearth/reearthx/rerror"
 	"github.com/reearth/reearthx/usecasex"
 )
 
 var (
 	ErrItemFieldRequired   = errors.New("item field required")
 	ErrInvalidField        = errors.New("invalid field")
-	ErrDuplicatedItemValue = errors.New("duplicated value")
+	ErrDuplicatedItemValue = rerror.NewE(&i18n.Message{ID: "item:duplicated value", Other: "The inserted value is duplicated"})
```

Comment:
```suggestion
	ErrDuplicatedItemValue = rerror.NewE(&i18n.Message{ID: "item:duplicated value"})
```

Default value is not needed because locale.en.yaml is used

Reply:
When I removed the default value the key will not be included in the generated files check this issue: https://github.com/nicksnyder/go-i18n/issues/245

Reply:
I think It is expected. You can add English messages to YAML, but not needed to add it to codes. You can use ID to recognize what message should be required.

Reply:
@rot1024 
But in the current behavior, I could not extract the messages which do not have a translation in the code using the `goi18n extract ...` command